sqlstore/src/main/resources/db/migration_afterMigrate/05_view_barcodable.sql

```diff
@@ -0,0 +1,43 @@
+CREATE OR REPLACE VIEW BarcodableView AS
+  SELECT
+    dilutionId AS targetId,
+    identificationBarcode,
+    name,
+    NULL       AS alias,
+    'DILUTION' AS targetType
+  FROM LibraryDilution
+  UNION ALL SELECT
+              poolId,
+              identificationBarcode,
+              name,
+              alias,
+              'POOL' AS targetType
+            FROM Pool
+  UNION ALL SELECT
+              sampleId,
+              identificationBarcode,
+              name,
+              alias,
+              'SAMPLE' AS targetType
+            FROM Sample
+  UNION ALL SELECT
+              libraryId,
+              identificationBarcode,
+              name,
+              alias,
+              'LIBRARY' AS targetType
+            FROM Library
+  UNION ALL SELECT
+              boxId,
+              identificationBarcode,
+              name,
+              alias,
+              'BOX' AS targetType
+            FROM Box
+  UNION ALL SELECT
+              containerId,
+              identificationBarcode,
+              NULL        AS name,
```

Comment:
It might make sense to have `identificationBarcode AS name` when you have to get around to displaying these things.

Reply:
Good point!  I'm not sure yet what these entries will look like in the frontend, but that may make sense.  I'm a little loathe to "misrepresent" the `name` of entities, though.

Any thoughts @djcooke ?